core/proto/keytransparency_v1_proto/keytransparency_v1_admin_proto.proto

```diff
@@ -54,9 +55,13 @@ message Domain {
   trillian.Tree map = 3;
   // Vrf contains the VRF public key.
   keyspb.PublicKey vrf = 4;
+  // min_interval is the minimum time between epochs. 
+  google.protobuf.Duration min_interval = 5;
+  // max_interval is the maximum time between epochs. 
+  google.protobuf.Duration max_interval = 6;
   // Deleted indicates whether the domain has been marked as deleted.
   // By its presence in a response, this domain has not been garbage collected.
-  bool deleted = 5;
+  bool deleted = 7;
```

Comment:
Usual warning about renumbering proto fields.  Maybe it's safe at this stage to do so.  But in general that's a data-compatibility breaking change.

Reply:
yep